contracts/CompliantDepositTokenWithHook.sol

```diff
@@ -25,10 +33,28 @@ contract CompliantDepositTokenWithHook is CompliantToken {
      * @param _value uint256 the amount of tokens to be transferred
      */
     function transferFrom(address _from, address _to, uint256 _value) public returns (bool) {
-        _transferFromAllArgs(_from, _to, _value, msg.sender);
+        if (uint256(_to) < REDEMPTION_ADDRESS_COUNT) {
+            registry.requireCanTransferFrom(msg.sender, _from, _to);
+            _value -= _value % CENT;
```

Comment:
This will still cause an exchange's accounting to be off. If I send 1000.005 to my redemption address, only 1000 will be sent.

Reply:
Better than stucking it in a redemption address

Reply:
Disagree, Why is it bad to have is stuck at redemption address, we could just round up the next time they redeem